hack/test-build.sh

```diff
@@ -28,9 +28,15 @@ if [ -z "$MY_QUAY_USER" ]; then
 else
   if oc get secret redhat-appstudio-staginguser-pull-secret &>/dev/null; then
      PUSH_WORKSPACE="-w name=registry-auth,secret=redhat-appstudio-staginguser-pull-secret"
+     # Ensure that the pipeline service account has access to the secret. Although the
+     # secret is mounted directly on the pipeline, Tekton Chains needs this linkage so
+     # it can push the image signature and attestation to the same OCI repository.
+     oc secrets link pipeline redhat-appstudio-staginguser-pull-secret --for=pull
```

Comment:
Does it error if you try to create an existing link? If not, then lgtm.

Reply:
It does not error.